docs/source/using/addons.adoc

```diff
@@ -171,7 +171,7 @@ $ minishift addons apply acme                                 // <2>
 If the environment variable is not set not, interpolation does not occur.
 <2> When the add-on is applied, each occurrence of `#{PROJECT_USER}` within an add-on command gets replaced with the value of the environment variable `USER`.
 
-As add-on developer, you can enforce that a variable value is provided when the add-on gets applied by adding the varaible name to the _Required-Vars_ metadata header.
+As add-on developer, you can enforce that a variable value is provided when the add-on gets applied by adding the variable name to the _Required-Vars_ metadata header.
```

Comment:
"As an add-on developer", although I actually think we can remove that statement entirely. The sentence reads perfectly well without it:

```adoc
You can enforce that a variable value is provided when....
```

Reply:
Done with "As an add-on developer".